transport-native-epoll/src/main/resources/META-INF/native-image/io.netty/channel/epoll/jni-config.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
should we add license headers or does this not work because its json ?

Reply:
JSON doesn't support comments. All our other `reflection-config.json` files also don't have any license headers.